DuckDuckGo/Preferences/View/PreferencesVPNView.swift

```diff
@@ -207,7 +207,7 @@ extension Preferences {
         /// Resolves the text to be used for exclusion counts
         ///
         private func exclusionCountString(value: Int) -> String {
-            value > 0 ? String(value) : "None"
+            value > 0 ? String(value) : UserText.vpnNoExclusionsFoundText
```

Comment:
One note on this, it didn't update automatically when I excluded an app, so it was telling me I had nothing excluded until I closed and re-opened the settings UI.

Reply:
Will try to get proper refresh fixed before merging this.

Reply:
Fixed.